### PR TITLE
feat: bump up cosmwasm/wasmvm v1.1.1

### PR DESCRIPTION
**Pull request by zemyblue**
# Description

* bump up cosmwasm/wasmvm v1.1.1

## Types of changes

- [x] Bug fix (changes which fixes an issue)
- [x] New feature (changes which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] ETC (build, ci, docs, perf, refactor, style, test)

## Checklist

- [ ] I followed the [contributing guidelines](https://github.com/line/wasmvm/blob/main/CONTRIBUTING.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] The PR title and commits are followed [conventional commit form](https://www.conventionalcommits.org/en/v1.0.0).

### CODE REVIEW

File: libwasmvm/Cargo.toml [hunk 1]
```diff
@@ -13,25 +13,25 @@ exclude = [".circleci/*", ".gitignore"]
...
 [features]
 default = []
 # This feature requires Rust nightly because it depends on the unstable backtrace feature.
 backtraces = []
 
 [dependencies]
-cosmwasm-std = { git = "https://github.com/line/cosmwasm", tag = "v1.0.0-0.6.0", features = ["iterator","staking","stargate"] }
-cosmwasm-vm = {  git = "https://github.com/line/cosmwasm", tag = "v1.0.0-0.6.0", features = ["iterator","staking","stargate"] }
+cosmwasm-std = { git = "https://github.com/line/cosmwasm", rev = "refs/pull/256/head", features = ["iterator","staking","stargate"]}
+cosmwasm-vm = { git = "https://github.com/line/cosmwasm", rev = "refs/pull/256/head", features = ["iterator","staking","stargate"]}
```
Comment (zemyblue) [inline libwasmvm/Cargo.toml:34]: I'll change this git path after merging https://github.com/line/cosmwasm/pull/256 PR.

File: libwasmvm/src/cache.rs [hunk 2]
```diff
@@ -719,18 +719,6 @@ mod tests {
             "0.2",
             "size_pinned_memory_cache: {size_pinned_memory_cache}"
         );
-        // todo check
```
Comment (loloicci) [inline libwasmvm/src/cache.rs]: why is it deleted?

Comment (zemyblue) [reply libwasmvm/src/cache.rs]: This codes is test code of wasmvm v1.0.0. When I bump up to wasmvm v1.1.1, this codes was conflicted, so I changed it to a comment. 
And at the final check, it was confirmed that it was removed the upstream repository of v1.1.1. So I delete it.

**Review by loloicci — APPROVED** [review]

**Review by shiki-tak — APPROVED** [review]
LGTM

**Pull request merged**